Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -461,6 +461,15 @@ extension TopTabsViewController: TabEventHandler {
             self.performTabUpdates()
         }
     }
+
+    func tab(_ tab: Tab, didChangeURL url: URL) {
+        assertIsMainThread("Animations can only be performed from the main thread")
```

Comment:
nit: not animation in this case; the bulk of our front-end code is not thread-safe